pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -460,7 +460,8 @@ func (opt *Options) Run(suite *TestSuite, junitSuiteName string) error {
 	if len(events) > 0 {
 		var buf *bytes.Buffer
 		syntheticTestResults, buf, _ = createSyntheticTestsFromMonitor(events, duration)
-		testCases := syntheticEventTests.JUnitsForEvents(events, duration, restConfig, suite.Name)
+		currResState := m.CurrentResourceState()
```

Comment:
I see some failures in CI:

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/27304/pull-ci-openshift-origin-master-e2e-gcp-upgrade/1556676643144077312

```
 pkg/test/ginkgo/cmd_runsuite.go:432:19: undefined: m
make: *** [vendor/github.com/openshift/build-machinery-go/make/targets/golang/build.mk:16: build] Error 2
cp: cannot stat '/go/src/github.com/openshift/origin/openshift-tests': No such file or directory 
```

I believe you need to rebase your PR on master.  `m` was removed in https://github.com/openshift/origin/commit/4e4c6d94189e7d688e389e824e9b8f5af3db45d0

Reply:
thanks.  I rebased, CI looked better.  I just squashed so CI will run again.